test/modes.js

```diff
@@ -276,13 +301,18 @@ test('Scribbletune::modes Miscellaneous', (t) => {
 		'Locrian #2 is also known as aeolian b5'
 	);
 	t.equal(
-		scribble.mode('c', 'major locrian', 3, false).join(''),
-		'c3d3e3f3f#3g#3a#3',
+		scribble.mode('c', 'major locrian', 4, false).join(''),
+		'c4d4e4f4f#4g#4a#4',
+		'Add root from next octave is customizable'
+	);
+	t.equal(
+		scribble.mode('c|major locrian|4|false').join(''),
+		'c4d4e4f4f#4g#4a#4',
```

Comment:
Unless I ve missed it, can we also have one test for `scribble.mode('c|major locrian|4')`?